build.sh

```diff
@@ -14,6 +14,7 @@ while [[ $# -gt 0 ]]
 do
 key="$1"
 case $key in
+    #TODO: Remove the shortform flag and update Jenkinsfile to use long-form
```

Comment:
This seems like it might be worth just doing?

Reply:
Done.